From ee3b68041ed9cf19b3dd18c4d1bb342abec7d2ac Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Tue, 9 Feb 2016 15:13:33 -0500
Subject: [PATCH] Add a workaround for {linux/net}/if.h conflict

Include linux/if.h and make sure we don't include the other file.

https://bugzilla.redhat.com/show_bug.cgi?id=1300256
---
 src/shared/firewall-util.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/shared/firewall-util.c b/src/shared/firewall-util.c
index 5acfb0191b..95335498ef 100644
--- a/src/shared/firewall-util.c
+++ b/src/shared/firewall-util.c
@@ -19,9 +19,11 @@
   along with systemd; If not, see <http://www.gnu.org/licenses/>.
 ***/
 
+#define _NET_IF_H 1
+
 #include <arpa/inet.h>
-#include <net/if.h>
 #include <sys/types.h>
+#include <linux/if.h>
 #include <linux/netfilter_ipv4/ip_tables.h>
 #include <linux/netfilter/nf_nat.h>
 #include <linux/netfilter/xt_addrtype.h>
-- 
1.9.2.649.g5ae4ba5

